src/components/MultiGestureCanvas/index.tsx

```diff
@@ -107,47 +115,65 @@ function MultiGestureCanvas({
     const offsetX = useSharedValue(0);
     const offsetY = useSharedValue(0);
 
+    useAnimatedReaction(
+        () => isSwipingDownToClose.value,
+        (current) => {
+            if (!isUsedInCarousel) {
+                return;
+            }
+            // eslint-disable-next-line react-compiler/react-compiler, no-param-reassign
+            isPagerScrollEnabled.value = !current;
+        },
```

Comment:
Fixed. In non-carousel, changing isScrollEnabled, changes the `nope` shared value here
https://github.com/Expensify/App/blob/64cb4ee92b1f4c66e5deb96e23caf47e25423c00/src/components/AttachmentModal.tsx#L474-L475

which sets `isPagerScrolling` to true too, disabling the pan gesture.